devspaces/aggregate/overlays/low-idle/kustomization.yaml

```diff
@@ -1,10 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: devspaces
-
 resources:
-  - ../../base
+  - ../default
```

Comment:
Instead of inheriting from the default overlay, I would suggest moving the default into a base and just have default be a simple reference to base.

The other overlays can then add whatever patches on top of base instead of default.